src/commands/generate/delta.ts

```diff
@@ -743,14 +780,24 @@ export default class GenerateDelta extends BaseCommand<typeof GenerateDelta> {
     let missingFlags = false
     let strMsg = 'Warning: The following errors occurred:\n'
 
-    if (!flags.inspecJsonFile) {
-      strMsg += colors.dim('  Missing required flag inspecJsonFile\n')
-      missingFlags = true
-    }
-
-    if (!flags.xccdfXmlFile) {
-      strMsg += colors.dim('  Missing required flag xccdfXmlFile\n')
-      missingFlags = true
+    // If we don't have a Controls Profiles summary file or are conducting
+    // a fuzzy matching we need the controls directory
+    if (!flags.inspecJsonFile || flags.runMapControls) {
+      // Check if the directory exists
+      if (fs.existsSync(flags.controlsDir)) {
+        const files = fs.readdirSync(flags.controlsDir)
+        // Filter the files to check if any of them have the .rd extension
```

Comment:
Do you mean `.rb`?

Reply:
yes, I used that for testing, great catch